engines/scumm/dialogs.cpp

```diff
@@ -962,6 +966,8 @@ LoomEgaGameOptionsWidget::LoomEgaGameOptionsWidget(GuiObject *boss, const Common
 	_overtureTicksValue->setFlags(GUI::WIDGET_CLEARBG);
 
 	_enableEnhancementsCheckbox = createEnhancementsCheckbox(widgetsBoss(), "LoomEgaGameOptionsDialog.EnableEnhancements");
+	_enableOriginalGUICheckbox = createOriginalGUICheckbox(widgetsBoss(), "LoomEgaGameOptionsDialog.EnableOriginalGUI");
+	_enableOriginalGUICheckbox->setVisible(false);  // TODO: remove when implemented
```

Comment:
Shouldn't this be removed?

Reply:
@AndywinXp is currently implementing all the original GUIs in PR [4328](https://github.com/scummvm/scummvm/pull/4328), the Loom EGA is not yet merged, so my idea was to put the control in place but hide it for now, as the checkbox does nothing.